src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -81,6 +82,37 @@ class CompanyStep extends React.Component {
             'password',
             'companyPhone',
         ];
+
+        // Keys in this.errorTranslationKeys are associated to inputs, they are a subset of the keys found in this.state
+        this.errorTranslationKeys = {
+            addressStreet: 'bankAccount.error.addressStreet',
+            addressCity: 'bankAccount.error.addressCity',
+            addressZipCode: 'bankAccount.error.zipCode',
+            companyName: 'bankAccount.error.companyName',
+            companyPhone: 'bankAccount.error.phoneNumber',
+            website: 'bankAccount.error.website',
+            companyTaxID: 'bankAccount.error.taxID',
+            incorporationDate: 'bankAccount.error.incorporationDate',
+            incorporationType: 'bankAccount.error.companyType',
+            industryCode: 'bankAccount.error.industryCode',
+            password: 'common.passwordCannotBeBlank',
+        };
+    }
+
+    /**
+     * @returns {Object}
+     */
+    getErrors() {
+        return lodashGet(this.props, ['reimbursementAccount', 'errors'], {});
+    }
+
+    /**
+     * @param {String} inputKey
+     * @returns {string}
```

Comment:
This should be capitalized https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs